### PR TITLE
Tackle-803: Maven configuration for MTA 6.0

### PR DESCRIPTION
**Pull request by CarmiWisemon**
MTA 6.0

Resolves: https://issues.redhat.com/browse/TACKLE-803

Preview: https://deploy-preview-563--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#mta-web-config-maven-rep_ocp-42

### CODE REVIEW

File: docs/topics/mta-web-config-maven-rep.adoc [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+= Configuring Maven repositories
+
+To configure a Maven repository or to clear the local artifact repository proceed with the following steps:
+
+== Configuring the repository
+
+.Procedure
+
+. In the left menu of the *Administrator* view, click *Repositories* and then *Maven*.
+. Click *Force update of dependencies* toggle switch to enable.
```
Comment (rromannissen) [inline docs/topics/mta-web-config-maven-rep.adoc:16]: I think this option has been removed. We need confirmation from @jortel

File: docs/topics/mta-web-config-maven-rep.adoc [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-maven-rep_{context}"]
+= Configuring Maven repositories
+
+To configure a Maven repository or to clear the local artifact repository proceed with the following steps:
```
[the remark this answers is not in the document]
Comment (CarmiWisemon) [reply docs/topics/mta-web-config-maven-rep.adoc]: @rromannissen I made the requested changes, please review.

File: docs/topics/mta-web-config-maven-rep.adoc (hunk 1 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-config-maven-rep.adoc:16]: @rromannissen waiting for a response.

Comment (jortel) [reply docs/topics/mta-web-config-maven-rep.adoc:16]: Yes. removed.

File: docs/topics/mta-web-config-maven-rep.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+= Configuring Maven repositories
+
+This view provides general options for the Maven repositories, such as managing the size of a local repository or configuring how to interact with external repositories.
+
+== Managing repository size
+
+.Procedure
+
+. In the left menu of the *Administrator* view, click *Repositories* and then *Maven*.
+. Click the *Clear repository* link.
```
Comment (istein1) [inline docs/topics/mta-web-config-maven-rep.adoc:16]: I would expect to have here elaboration on the maven repositories options mentioned right above, but it is not.
It's just saying click "Clear repository"

Comment (CarmiWisemon) [reply docs/topics/mta-web-config-maven-rep.adoc:16]: @rromannissen @jortel  is there any other information we should beyond what's written above?

File: docs/topics/mta-web-config-maven-rep.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+[id="mta-web-config-maven-rep_{context}"]
+= Configuring Maven repositories
+
+This view provides general options for the Maven repositories, such as managing the size of a local repository or configuring how to interact with external repositories.
+
+== Managing repository size
+
+.Procedure
+
+. In the *Administrator* view, click *Repositories* and then *Maven*.
```
Comment (RichardHoch) [inline docs/topics/mta-web-config-maven-rep.adoc:15]: Better to repeat "click" -- that is.

**Comment by RichardHoch** [comment]
One small note. Other than that, LGTM.

File: docs/topics/mta-web-config-maven-rep.adoc (hunk 4 above)
Comment (CarmiWisemon) [reply docs/topics/mta-web-config-maven-rep.adoc]: In the Administrator view, click Repositories and then click Maven.

**Pull request merged**